src/richie-front/js/components/SearchFilter/_SearchFilter.scss

```diff
@@ -19,6 +19,7 @@ $richie-search-filters-active-fontweight: null !default;
 $richie-search-filters-active-background: null !default;
 $richie-search-filters-active-divider-border: null !default;
 
+$richie-search-filters-close-font-size: $font-size-base * 2 !default;
```

Comment:
I think we can make it more generic, _i.e._ `$richie-search-filters-icon-font-size`. WDYT?

Reply:
why not but doing this, every new icon added in the search interface will have this size which is pretty big. Is it what you want ? I though the close icon is an exception but if it's not the case we can change this name yes.

Reply:
I'll let @mbenadda answer this question, but IMO, if the close icon is really bigger, we can do something like:

```scss
&__close {
  // [...]
  font-size: 1.5 * $richie-search-filters-icon-font-size;
}
```